.storybook/itwinTheme.js

```diff
@@ -51,7 +51,7 @@ export const darkTheme = create({
 
   // UI
   appBg: '#2D373C',
-  appContentBg: '#4F5D65',
+  appContentBg: 'rgb(90, 105, 115)', // background-1
```

Comment:
hex would look better....

Reply:
Didnt need to change for both themes?

Reply:
This is for Docs tab. Previous background looked really bad in Dark theme, but was ok in Light theme.

Reply:
Its not only for docs..its for main too, from what I remember

Reply:
Yeah but it doesn't look that bad in main tab.

Light theme:
![image](https://user-images.githubusercontent.com/9084735/126528197-3bce7b05-7b97-4163-ac8a-8df594305ac5.png)

Dark theme Docs (current):
![image](https://user-images.githubusercontent.com/9084735/126528209-916bf437-ae1a-421e-abaa-63d5a8bfe0ba.png)

Dark theme Docs (previous) notice lack of contrast:
![image](https://user-images.githubusercontent.com/9084735/126528665-28a24bc1-58da-4c21-8e0f-9840ea6de666.png)

Dark theme Canvas (current):
![image](https://user-images.githubusercontent.com/9084735/126528341-1a03c5d1-4b74-4d37-9401-c13c6d7a8078.png)

Dark theme Canvas (previous):
![image](https://user-images.githubusercontent.com/9084735/126528461-13af25af-d0e3-4243-898d-b30f56afb6ce.png)